app/assets/scss/_edit_section.scss

```diff
@@ -0,0 +1,5 @@
+.question-hint {
```

Comment:
Addressed in: https://github.com/alphagov/digitalmarketplace-admin-frontend/commit/b6ab578e4c604420b8ef8bf43314add312061441